benchbuild/projects/polybench/polybench.py

```diff
@@ -156,11 +160,14 @@ def filter_stderr(stderr_raw, stderr_filtered):
             noopt_stderr_filtered = noopt_stderr_raw + ".filtered"
 
             with local.env(BB_IS_BASELINE=True):
-                runner(wrapping.wrap(binary, self))
+                polybench_bin = wrapping.wrap(binary, self)
+                polybench_bin = run.watch(polybench_bin)
+                polybench_bin()
             filter_stderr(noopt_stderr_raw, noopt_stderr_filtered)
 
-            diff_cmd = diff[noopt_stderr_filtered, opt_stderr_filtered]
-            runner(diff_cmd, retcode=0)
+            diff_ = diff[noopt_stderr_filtered, opt_stderr_filtered]
+            diff_ = run.watch(diff_)
+            diff_(retcode=0)
```

Comment:
Even in the same file it is different :laughing:

Reply:
Fixed in cc611b4a00e299b1eed8084f79553c9b3259e800